src/IlkRegistry.sol

```diff
@@ -332,16 +370,68 @@ contract IlkRegistry {
         (address _pip,) = spot.ilks(ilk);
         require(_pip != address(0), "IlkRegistry/pip-invalid");
 
-        (address _flip,,) = cat.ilks(ilk);
-        require(_flip != address(0), "IlkRegistry/flip-invalid");
-        require(FlipLike(_flip).cat() == address(cat), "IlkRegistry/flip-wrong-cat");
-        require(FlipLike(_flip).vat() == address(vat), "IlkRegistry/flip-wrong-vat");
+        (address _clip,,,) = dog.ilks(ilk);
```

Comment:
Maybe the public update function should just be removed entirely, since it can override governance-induced updates...

Reply:
Or we keep ability to update pip and not [f/c]lip